x/group/spec/messages.md

```diff
@@ -0,0 +1,46 @@
+```go
+// Groups get their own GroupID
+type MsgCreateGroup struct {
+	Signer sdk.AccAddress `json:"signer"`
+	// The Owner of the group is allowed to change the group structure. A group account
+	// can own a group in order for the group to be able to manage its own members
+	Owner  sdk.AccAddress `json:"owner"`
+	// The members of the group and their associated weight
+	Members []Member `json:"members,omitempty"`
+	// TODO maybe make this something more specific to a domain name or a claim on identity? or Info leave it generic
+	Memo string `json:"memo,omitempty"`
```

Comment:
If we store this on the group itself, I would call it `Description` or `Title`.
I think that makes sense, so they can identify themselves.
I do think storing this makes sense.

If it is just a memo for the transaction itself, we don't need it imo. There is a top-level one.

Reply:
`Description` then?

Reply:
Agree, memo is used already in a different context. For UX purpose in clients it may be very useful to store human readable information with a group. Title and description both make sense.